### PR TITLE
Wazo 3350 - Remove queueskill category

### PR DESCRIPTION
**Pull request by xxbillouxx**
Depends-on: https://github.com/wazo-platform/xivo-dao/pull/248

### CODE REVIEW

**Review by DrPyser — CHANGES_REQUESTED** [review]
https://github.com/wazo-platform/wazo-confd/blob/8204008a1f15b44f6ae04dda5a8ececa27d39a58/wazo_confd/plugins/skill/schema.py#L16
The schema field for category should be deleted if the API doesn't expose/support that field anymore.

**Review by DrPyser — CHANGES_REQUESTED** [review]
Could you update the CHANGELOG.md since this is a breaking change in the public API?

**Comment by xxbillouxx** [comment]
> Could you update the CHANGELOG.md since this is a breaking change in the public API?

Are you ok with that: `the "agent skill" resource now has a no more "category" field` ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
   * POST `/1.1/contexts/<context_id>/ranges/<range type>`. It can be used to list available extensions in a given context.
 
+* the `agent skill` resource now has no more `category` field.
```
Comment (DrPyser) [inline CHANGELOG.md:9]:
```suggestion
* the `agent skill` resource no longer has a `category` field.
```

**Review by DrPyser — DISMISSED** [review]
Well done!

**Review by DrPyser — APPROVED** [review]

**Pull request merged**